components/List.js

```diff
@@ -0,0 +1,32 @@
+import styled from "styled-components";
+import { nowPlayingMovies } from "../lib/db";
+import MovieCard from "./Card";
+
+const StyledList = styled.li`
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+`;
+
+export default function NowPlayinglist() {
+  return (
+    <>
+      <h3>Now Playing</h3>
+
+      <ul>
+        <StyledList>
+          {nowPlayingMovies.map((movie) => (
+            <li key={movie.id}>
+              <MovieCard
+                image={movie.image}
+                year={movie.year}
+                title={movie.title}
+              />
+            </li>
+          ))}
+        </StyledList>
```

Comment:
you shouldn't have several list items as the child of another list item.

Better to write it like this:

```suggestion
          {nowPlayingMovies.map((movie) => (
            <StyledList key={movie.id}>
              <MovieCard
                image={movie.image}
                year={movie.year}
                title={movie.title}
              />
            </StyledList>
          ))}
```